sumOfOther.js

```diff
@@ -0,0 +1,7 @@
+function sumOfOther(array) {
+
+let sum = array.reduce((acc, item)=> (acc+item));
```

Comment:
It's better always to add initial value to reduce
arr.reduce(callback[, initialValue])